extensions/amp-ad-network-fake-impl/0.1/amp-ad-network-fake-impl.js

```diff
@@ -60,9 +58,8 @@ export class AmpAdNetworkFakeImpl extends AmpA4A {
           // `localDev` and primarily used for A4A Envelope for testing.
           // See DEVELOPING.md for more info.
           const creative = this.transformCreativeLocalDev_(deserialized);
-          const encoder = new TextEncoder('utf-8');
           return {
-            creative: encoder.encode(creative).buffer,
+            creative: utf8EncodeSync(creative),
             signature: 'FAKESIG',
```

Comment:
Looking at https://github.com/ampproject/amphtml/blob/master/src/utils/bytes.js#L84 in the case where TextEncoder is defined, utf8EncodeSync(creative) is equivalent to encoder.encode(creative), not encoder.encode(creative).buffer.
If it's easy, e.g. not too many other uses, I'd suggest changing utf8EncodeSync to return .buffer in this case (line 86). But if that would cause problems to unrelated code, then it looks like this code here would have to change to something like foo=utf8EncodeSync(creative); foo = foo.buffer || foo; return { creative:foo, ... }